internal/s3select/select.go

```diff
@@ -548,6 +548,17 @@ OuterLoop:
 				}
 
 				outputQueue[len(outputQueue)-1] = outputRecord
+				if s3Select.statement.LimitReached() {
+					if !sendRecord() {
+						break
+					}
+					if err = writer.Finish(s3Select.getProgress()); err != nil {
+						// FIXME: log this error.
```

Comment:
why not log this error now itself

Reply:
It is not decided should we even log @kannappanr